### PR TITLE
9344 scaffold view payments

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9344)

This ticket is to set up the basic structure for the View Payments page. 

## Testing done
Unit tests will be included in a separate ticket

## Screenshots
[screencapture-localhost-3001-disability-benefits-view-payments-2020-05-21-12_37_29.pdf](https://github.com/department-of-veterans-affairs/vets-website/files/4664363/screencapture-localhost-3001-disability-benefits-view-payments-2020-05-21-12_37_29.pdf)

## Acceptance criteria
- [x] Create the pages for the various screens for this feature
- [x] Frame or stub out the component areas based on the prototype / mockup
- [x] Create a list of next steps that need to be implemented

### CODE REVIEW

File: src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlockStates/ViewPaymentsSidebarBlockStates.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export const secondSidebarBlock = {
+  heading: 'What if I’m missing a payment?',
+  content: (
+    <>
+      <p>
+        Please wait at least 3 business days (Monday-Friday) before reporting
+        non-receipt of a payment as check tracing cannot be initiated before
+        that period. To report non-receipt after 3 business days, call
+        <a href="tel:8008271000" aria-label="1. 800. 827. 1000s">
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlockStates/ViewPaymentsSidebarBlockStates.jsx:26]: I think the 's' on the end of 1000 can be removed

File: src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlockStates/ViewPaymentsSidebarBlockStates.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    </>
+  ),
+};
+
+export const thirdSidebarBlock = {
+  heading: 'What if I have questions?',
+  content: (
+    <>
+      <p>
+        You can call us at <a href="tel:8008271000">800-827-1000</a>. We're here
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlockStates/ViewPaymentsSidebarBlockStates.jsx:41]: This should include an aria-label, similar to the number above.

File: src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ViewDependentsSidebar = props => (
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebar.jsx:3]: This should be ViewPaymentsSidebar

File: src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlock.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const ViewDependentsSidebarBlock = props => (
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/ViewPaymentsSidebar/ViewPaymentsSidebarBlock.jsx:3]: This should be ViewPaymentsSidebarBlock

File: src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PaymentsReceived from './payments-recieved/PaymentsReceived.jsx';
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx:2]: Should this import path be spelled `./payments-received/...`?

File: src/applications/disability-benefits/view-payments/components/view-payments-lists/payments-recieved/PaymentsReceived.jsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/view-payments-lists/payments-recieved/PaymentsReceived.jsx:1]: This directory should be named `payments-received`

File: src/applications/disability-benefits/view-payments/containers/helpers.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const breadcrumbLinks = [
+  <a href="/" aria-label="Back to VA Home page" key="1">
+    Home
+  </a>,
+  <a
+    href="/view-change-dependents/"
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/containers/helpers.js:8]: Should this href hold a different value?

File: src/applications/disability-benefits/view-payments/containers/helpers.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    aria-label="View your VA payment history"
+    key="3"
+  >
+    View your VA payment history
+  </a>,
+  <a href="/view-change-dependents/view" key="4">
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/containers/helpers.js:14]: Should this href hold a different value?

**Review by micahchiang — CHANGES_REQUESTED** [review]
Requesting changes for some of the component names.

File: src/applications/disability-benefits/view-payments/containers/helpers.js (hunk 7 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/view-payments/containers/helpers.js:8]: Yes but I don't know what the hrefs should be yet, they are still working on an IA

File: src/applications/disability-benefits/view-payments/containers/helpers.js (hunk 8 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/view-payments/containers/helpers.js:14]: Same as above

**Review by micahchiang — APPROVED** [review]
lgtm

**Pull request merged**